cms_form/static/src/js/date_widget.js

```diff
@@ -1,45 +1,77 @@
 odoo.define('cms_form.date_widget', function (require) {
   'use strict';
 
-  require('web.dom_ready');
+  var sAnimation = require('website.content.snippets.animation');
+  var time_utils = require('web.time');
 
-  // TODO: migrate to snippet animation
-  $(document).ready(function () {
-    $("input.js_datepicker").each(function () {
-      var $input = $(this);
-      var options = _.defaults(
-        $input.data('params') || {}, {
-          useSeconds: false,
+  sAnimation.registry.CMSDateWidget = sAnimation.Class.extend({
+    selector: ".cms_form_wrapper form input.js_datepicker",
+    start: function () {
```

Comment:
shouldn't such configuration be placed in `willStart` method?

Reply:
not necessarily